### PR TITLE
1464 Improve English Translations

### LINKED ISSUE #1464: Improve English Translations
Some English translations use poor grammar or improper words. This ticket is a placeholder so I can submit a pull request to improve the English translation.

### PR DESCRIPTION
**Pull request by jlefler**
This pull request has no functional changes. It merely updates the English translation with minor changes to word choice and grammar to improve the user experience for English-speaking users. The incorrect translation that prompted the change was 'Actualise' when performing a feed update. I have replaced that with "Updating" and made other similar changes. Fixes #1464.

### CODE REVIEW

File: app/i18n/en/conf.php [hunk 1]
```diff
@@ -142,7 +142,7 @@
 		'_' => 'Shortcuts',
 		'article_action' => 'Article actions',
 		'auto_share' => 'Share',
-		'auto_share_help' => 'If there is only one sharing mode, it is used. Else modes are accessible by their number.',
+		'auto_share_help' => 'If there is only one sharing mode, it is used. Other modes are accessible by their number.',
```
Comment (Alwaysin) [inline app/i18n/en/conf.php]: I think that "else" was used as in "otherwise" here - not "other", which would make more sense.

**Review by Alwaysin — CHANGES_REQUESTED** [review]

**Comment by jlefler** [comment]
I'm fine with otherwise rather than other here.

File: app/i18n/en/admin.php [hunk 2]
```diff
@@ -138,7 +138,7 @@
 		'number_entries' => '%d articles',
 		'percent_of_total' => '%% of total',
 		'repartition' => 'Articles repartition',
-		'status_favorites' => 'Favourites',
+		'status_favorites' => 'Favorites',
```
Comment (aledeg) [inline app/i18n/en/admin.php]: Both are correct. One is for US, the other one is for UK. I have no preference even though I am more used to "Favourites"

**Comment by aledeg** [comment]
Look good to me.

@jlefler thanks

**Comment by Alkarex** [comment]
Merged. Thanks again!

File: app/i18n/en/gen.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 return array(
 	'action' => array(
-		'actualize' => 'Actualize',
+		'actualize' => 'Updating',
```
Comment (Alkarex) [inline app/i18n/en/gen.php:5]: I think this one is wrong:
image
Should be something like *Actualize* or *Update*

Comment (Alkarex) [reply app/i18n/en/gen.php:5]: Fixed in https://github.com/FreshRSS/FreshRSS/pull/1492

**Comment by Frenzie** [comment]
I would also change the "damn!" in error messages to something slightly more proper even if it's only a stylistic issue.

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/pull/1561

**Pull request merged**